### DISCUSSION TITLE
how to assign ssl to localhost easily?

### DISCUSSION THREAD

**Discussion by skinkairewalker** [uNetworking/uWebSockets#1700] [Q&A]
Hello everyone, I would like to know how to assign ssl on localhost while testing with and without ssl my http server.
Would this be possible?

**Comment by uNetworkingAB** [comment]
You can just clone the repo, hit `WITH_OPENSSL=1 make` and try HelloWorld with curl --insecure https://localhost:3000 or whatever the port may be.

**Reply by skinkairewalker** [reply]
awesome :D
Thanks !
